File: MaxProductOfThree/Python/MaxProductOfThree.py
def maxProductOfThree(someList):
    temp = sorted(someList, reverse = True)
    return max(temp[0] * temp[1] * temp[2], temp[0] * temp[-1] * temp[-2])

listOne = [-3, 1, 2, -2, 5, 6]
print(maxProductOfThree(listOne))

File: MaxProductOfThree/C#/MaxProductOfThree.cs
using System;

namespace MaxProductOfThree
{
	class Program
	{
		public static int MaxProductOfThree(int [] someArr)
		{
			Array.Sort(someArr);
			
			//Or you can do without reversing but changing the return	
			Array.Reverse(someArr);
		
			return Math.Max(someArr[0] * someArr[1] * someArr[2],
					someArr[0] * someArr[someArr.Length - 1] * someArr[someArr.Length - 2]);
		}

		public static void Main(string[] args)
		{
			int [] arrOne = {-3, 1, 2, -2, 5, 6};
			Console.WriteLine(MaxProductOfThree(arrOne));
		}
	}
}
